frontend/src/components/regions/UserActions/utils.js

```diff
@@ -0,0 +1,61 @@
+export function updateBurnDownSnapshots({ burnDownSnapshots }) {
+  const datesArray = [];
+  burnDownSnapshots.forEach((burnDownSnapshot) => {
+    datesArray.push(burnDownSnapshot.timestamp);
+  });
```

Comment:
This is good. I just want to guide you to writing code more efficiently. It'll save you time in the long run

Try using a `map`  to get the datesArray. Where else can map be useful?

Reply:
Have you tried to implement sheena's suggestion here? its not in the current changes if so